Client/Frontend/AuthenticationManager/AuthenticationSettingsViewController.swift

```diff
@@ -157,7 +157,7 @@ class TouchIDSetting: Setting {
         let accessoryContainer = UIView(frame: control.frame)
         accessoryContainer.addSubview(control)
 
-        let gesture = UITapGestureRecognizer(target: self, action: "switchTapped")
```

Comment:
Doesn't a later commit in this patch cover this? I don't see a warning for this one anymore.